package.json

```
@@ -30,7 +30,7 @@
 		"clean-jetpack": "cd projects/plugins/jetpack && yarn clean",
 		"distclean": "rm -rf node_modules && yarn clean-composer && yarn distclean-jetpack",
 		"distclean-jetpack": "cd projects/plugins/jetpack && yarn distclean",
-		"docker:build": "docker run -it --rm  -v ${PWD}:/usr/src/app -w /usr/src/app node:12 yarn build",
+		"docker:build": "docker run -it --rm  -v ${PWD}:/usr/src/app -w /usr/src/app node:$(cat .nvmrc) yarn build",
```

Might be worth quoting it, just in case.
```suggestion
		"docker:build": "docker run -it --rm -v \"${PWD}:/usr/src/app\" -w /usr/src/app \"node:$(cat .nvmrc)\" yarn build",
```

---

Without objection.